src/CMakeLists.txt

```diff
@@ -37,16 +38,19 @@ else()
 endif()
 
 string(APPEND OUTPUT_DIR "${PROJECT_SOURCE_DIR}/binary")
+
 if(CMAKE_SYSTEM_NAME STREQUAL "Windows")
     string(APPEND OUTPUT_DIR "/windows")
 elseif(CMAKE_SYSTEM_NAME STREQUAL "Linux")
     string(APPEND OUTPUT_DIR "/linux")
 elseif(CMAKE_SYSTEM_NAME STREQUAL "Darwin")
     string(APPEND OUTPUT_DIR "/macos")
+    add_cxx_flag_if_supported(-fno-objc-msgsend-selector-stubs)
 elseif(CMAKE_SYSTEM_NAME STREQUAL "Android")
     string(APPEND OUTPUT_DIR "/android/${CMAKE_ANDROID_ARCH_ABI}")
 elseif(CMAKE_SYSTEM_NAME STREQUAL "iOS")
     string(APPEND OUTPUT_DIR "/ios")
+    add_cxx_flag_if_supported(-fno-objc-msgsend-selector-stubs)
```

Comment:
I would add these later in the 
`elseif(CMAKE_SYSTEM_NAME STREQUAL iOS)`
and would add another branch for macOS as 
`elseif(CMAKE_SYSTEM_NAME STREQUAL "Darwin")`

Reply:
Ok, I will return it backwards.